metadata_json_deps.gemspec

```diff
@@ -12,8 +12,9 @@ Gem::Specification.new do |s|
   s.metadata    = { 'source_code_uri' => 'https://github.com/ekohl/metadata_json_deps' }
   s.executables << 'metadata-json-deps'
 
-  s.required_ruby_version = Gem::Requirement.new('>= 2.0.0')
+  # puppet_forge requires Ruby 2.4
+  s.required_ruby_version = Gem::Requirement.new('>= 2.4.0')
```

Comment:
I don't mind this, but please keep https://github.com/ekohl/metadata_json_deps#compatibility in sync.

Reply:
updated it